test/es6module/bug_OS12095746.baseline

```diff
@@ -1,6 +1,3 @@
-NotifyModuleReadyCallback(exception) bug_OS12095746_mod0.js
-NotifyModuleReadyCallback(exception) bug_OS12095746_mod1.js
 mod0 catch:Syntax error
 mod1 catch:Unexpected identifier after numeric literal
-NotifyModuleReadyCallback(exception) bug_OS12095746_mod2.js

```

Comment:
This callback no longer fires for Dynamic module exceptions - there was no useful purpose to it firing before as Host ought to disregard it and leaving handling to within script - as promise is rejected.